.github/ISSUE_TEMPLATE.md

```diff
@@ -11,5 +11,11 @@
 ### Steps to reproduce:
 <!-- If you can then please provide the steps to reproduce the bug -->
 
+### Configurations:
```

Comment:
Is "Configurations" a valid English word? Can we not use "Environment" here instead?

Reply:
I thought of "Environment" before but was not sure it will be appropriate or not.
will change it to "Environment".